server/database/createSeed.js

```diff
@@ -1,8 +1,13 @@
+//1m authors, 10m reviews, 10mrestaurants, 1k pictures
```

Comment:
Consider renaming, the purpose of this script isn't to seed the database, it is to generate the data needed to seed the database.

Reply:
Hmm any suggestions? I thoguht 'createseed' was pretty clear as it's creating the 'seed' in the sense of RNG, the randomization basically. I mean, I have the actual faker seed set to be constant. But that means this 'seed' for the database is good. I don't know, maybe the term seed means something different for databases then for pRNG and I'm confused in which case I'll take suggestions.